libraries/AP_Mount/STorM32_lib.cpp

```diff
@@ -334,7 +338,11 @@ void STorM32_lib::receive_reset(void)
 
 void STorM32_lib::receive_reset_wflush(void)
 {
-    while (_serial_available() > 0) {
+    //it is claimed a while can't do it, so do a for
+    for (uint32_t i = 0; i < _serial_available(); i++) {
```

Comment:
Why do you think this is OK? `_serial_available` will be called in every loop and that's the issue, it might take a long time to be less or equal than `i`.